package.json

```diff
@@ -7,7 +7,9 @@
     "clean": "lerna clean && lerna run clean",
     "bootstrap": "lerna bootstrap",
     "publish": "npm run build && lerna publish",
-    "build": "lerna run build",
+    "build": "npm run build:cjs && npm run build:esm",
+    "build:cjs": "lerna run build --stream",
+    "build:esm": "rollup -c",
```

Comment:
rollup script runs at the top level. this makes for a pretty fast build process, and makes sure that things are built in the right order. it doesn't do type-checker, so we leave that to our existing cjs build process, which is why that one should always run first.